Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -798,14 +798,34 @@ def perform_rasterize(path: str | list[str],
     :param width: window width
     :param height: window height
     """
-    demisto.debug(f"perform_rasterize, {path=}, {rasterize_type=}")
+
+    # convert the paths param to list in case we have only one string
+    paths = argToList(paths)
```

Comment:
From here on you know it's an array.
```suggestion
    paths = argToList(path)
```

Reply:
done